e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/filter-payment-details/filter-payment-details.spec.js

```diff
@@ -166,5 +184,124 @@ context(`users can filter payment details by facility id and payment reference a
       paymentDetailsTable.row(fourthPayment.id, thirdFeeRecord.id).should('not.exist');
       paymentDetailsTable.row(fourthPayment.id, fourthFeeRecord.id).should('not.exist');
     });
+
+    it('should display all applied filters as selected filters', () => {
+      const paymentCurrencyFilter = CURRENCY.GBP;
+      const paymentReferenceFilter = 'payment ref';
+      const facilityIdFilter = '1111';
+
+      const feeRecord = aFeeRecordWithIdAndFacilityId(1, '11111111');
+      const payment = aPaymentWithIdFeeRecordsAndReference(11, [feeRecord], 'ABCD');
+      cy.task(NODE_TASKS.INSERT_PAYMENTS_INTO_DB, [payment]);
+
+      const matchingTfmFacilities = getMatchingTfmFacilitiesForFeeRecords([feeRecord]);
+      cy.task(NODE_TASKS.INSERT_TFM_FACILITIES_INTO_DB, matchingTfmFacilities);
+
+      pages.landingPage.visit();
+      cy.login(USERS.PDC_RECONCILE);
+
+      cy.visit(`/utilisation-reports/${reportId}`);
+
+      paymentDetailsTabLink().click();
+
+      filters.paymentCurrencyRadioInput(paymentCurrencyFilter).click();
+      cy.keyboardInput(filters.paymentReferenceInput(), paymentReferenceFilter);
+      cy.keyboardInput(filters.facilityIdInput(), facilityIdFilter);
+
+      filters.submitButton().click();
+
+      filters.panel().should('contain', 'Selected filters');
+
+      filters.panel().within(() => {
+        cy.assertText(cy.get('h3').eq(0), 'Currency');
+        cy.assertText(cy.get('h3').eq(1), 'Payment reference');
+        cy.assertText(cy.get('h3').eq(2), 'Facility ID');
+      });
+
+      cy.assertText(filters.selectedFilter(paymentCurrencyFilter), `Remove this filter ${paymentCurrencyFilter}`);
+      cy.assertText(filters.selectedFilter(paymentReferenceFilter), `Remove this filter ${paymentReferenceFilter}`);
+      cy.assertText(filters.selectedFilter(facilityIdFilter), `Remove this filter ${facilityIdFilter}`);
+
+      cy.assertText(filters.actionBarItem(facilityIdFilter), `Remove this filter ${facilityIdFilter}`);
+      cy.assertText(filters.actionBarItem(paymentCurrencyFilter), `Remove this filter ${paymentCurrencyFilter}`);
+      cy.assertText(filters.actionBarItem(paymentReferenceFilter), `Remove this filter ${paymentReferenceFilter}`);
+    });
+
+    it('should remove all filters when clear filters button clicked', () => {
```

Comment:
There's a lot going on in each "it", could we split into "its" and "describess" so it's easier to read/update in the future? E.g something like this

```js
describe('when clear filters button clicked', () => {
  before(() => {
    // login, click XYZ etc
  });

  it('should remove all filters', () => {
    // assertions
  });
});
```